### PR TITLE
added people and fixed landmark class

### PR DESCRIPTION
**Pull request by jenniferstarkey**
fixed landmarkS class issue and added list of people.

### CODE REVIEW

File: thailand.html [hunk 1]
```diff
@@ -37,10 +37,22 @@ <h2>Thailand</h2>
         <section class="topFlex">
             <div class="landmarks">
                 <h3>Landmarks</h3>
-                <img src="images/wat-arun.jpg" alt="Wat Arun" class="">
-                <img src="images/wat-phra-kaew.jpg" alt="Wat Phra Kaew" class="">
-                <img src="images/wat-phra-that-doi-suthep.jpg" alt="Wat Phrathat Doi Suthep" class="">
-                <img src="images/ko-tapu.jpg" alt="Ko Tapu" class="">
```
Comment (HanakoH) [inline thailand.html:43]: You might want to add <figure> and <figcaption>

Comment (HanakoH) [reply thailand.html:43]: < figure > and < figcaption >

File: thailand.html [hunk 2]
```diff
@@ -18,7 +18,7 @@ <h1>Lúcuma Travel Service</h1>
                 <div class="navDiv">
                     <nav>Places We Offer:
                     <ul class="nav">
-                        <li class="button button1"><a href="chile.html">Chile</a></li>
```
Comment (HanakoH) [inline thailand.html:21]: Please change this to index

**Comment by HanakoH** [comment]
DOOO EET

**Review by blaker814 — APPROVED** [review]
Yay! Changes seemed to work

**Review by HanakoH — APPROVED** [review]

**Pull request merged**